magicgui/widgets/_bases/create_widget.py

```diff
@@ -20,6 +20,7 @@ def create_widget(
     label=None,
     gui_only=False,
     app=None,
+    is_input: bool = True,
```

Comment:
see other comment, move after `options` and change to `is_result=False`

Reply:
I made the suggested change and rebased it into the commit that added this so that we wouldn't have any commits where `is_input` existed. Please let me know what you think.